api/src/opentrons/protocol_api/labware.py

```diff
@@ -317,9 +325,30 @@ def uri(self) -> str:
 
     @property  # type: ignore[misc]
     @requires_version(2, 0)
-    def parent(self) -> LocationLabware:
+    def parent(self) -> Optional[Union[str, ModuleTypes]]:
```

Comment:
Very small style suggestion: put `None` in the `Union` rather than using `Optional`

```suggestion
    def parent(self) -> Union[str, ModuleTypes, None]:
```